lib/localeapp/cli/install.rb

```diff
@@ -79,13 +83,25 @@ def write_config_file
           write_rails_config
         end
 
+        def write_dotenv
+          @output.puts "NOTICE: .env detected. Your API key was saved there."
+          File.open('.env', 'a') do |file|
+            file.puts
+            file.puts "LOCALEAPP_API_KEY=#{self.key}"
+          end
+        end
+
+        def config_api_key_str
+          self.dotenv ? "ENV['LOCALEAPP_API_KEY']" : "'#{key}'"
```

Comment:
I know, I was attempting to accommodate to style of the surrounding code.

Explicit receiver `self` removed.